apps/nextjs-website/src/components/organisms/Auth/PasswordTextField.tsx

```diff
@@ -32,35 +32,40 @@ export const PasswordTextField = ({
     event.preventDefault();
   };
 
+  const endAdornment = (
```

Comment:
This is calculated every rerender, it should be put in a dedicated component or memoized
```suggestion
  const endAdornment = useMemo ( () => (
      <InputAdornment position='end'>
      <IconButton
        aria-label='toggle password visibility'
        onClick={handleClickShowPassword}
        onMouseDown={handleMouseDownPassword}
        edge='end'
      >
        {showPassword ? <VisibilityOff /> : <Visibility />}
      </IconButton>
    </InputAdornment>), [showPassword, handleClickShowPassword, handleMouseDownPassword ])
```